server/routes/caseAdmin/assessment.ts

```diff
@@ -11,14 +12,45 @@ export default class AssessmentRoutes {
       res.locals.agent,
       req.params.prisonNumber,
     )
-    const { status } = assessmentSummary
 
+    const assessmentContactsResponse = await this.caseAdminCaseloadService.getCurrentAssessmentContactDetails(
+      req?.middleware?.clientToken,
+      res.locals.agent,
+      req.params.prisonNumber,
+    )
+
+    const { status } = assessmentSummary
     res.render('pages/caseAdmin/assessment', {
       assessmentSummary: {
         ...assessmentSummary,
         tasks: assessmentSummary.tasks.PRISON_CA,
+        contactList: this.createContactData(assessmentContactsResponse ? assessmentContactsResponse.contacts : null),
```

Comment:
mmh..`assessmentContactsResponse` will never be missing right?  
At the mo, the client would return an empty object as a bare minimum which is truthy

I'd suggest pushing uncertainty around presence of objects into the boundaries and not be so defensive in the core of the app.. 

Happy for 404 to error, which is the default behaviour 
But ideally the backend API should be returning an empty array if no contacts
The client would then return an empty array.
So here we don't need to handle possible undefined vars

Reply:
yeha defo contacts would be []